content/src/mappings/callout-alerts.json

```diff
@@ -0,0 +1,10 @@
+{
+  "calloutAlerts": {
+    "calloutAlertGroupTitle": "${alertVariant} alert",
+    "infoOverride": "informational",
+    "successOverride": "success",
+    "errorOverride": "error",
+    "warningOverride": "warning",
+    "noteOverride": "note"
```

Comment:
[sand] Is "override" the right name for these? It feels like this is just being used to set the type, less override a default value.